static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -815,11 +830,17 @@ function WidgetBuilder({
     DisplayType.BIG_NUMBER,
   ].includes(state.displayType);
 
-  const {columns, aggregates, fields} = state.queries[0];
-  const explodedColumns = columns.map(field => explodeField({field}));
-  const explodedAggregates = aggregates.map(field => explodeField({field}));
+  const {columns, aggregates, fields, fieldAliases = []} = state.queries[0];
```

Comment:
If state.queries is [] this will crash, also why do we just take state.queries[0] here? Maybe worth explaining

Reply:
`state.queries[0]` will never be empty because we don't allow the user to delete the last query. it's always [0] because a table visualization can only have one query